qCC/ccEntityAction.cpp

```diff
@@ -809,10 +810,10 @@ namespace ccEntityAction
 					8,
 					nullptr,
 					parent);
-				dlg.setWindowTitle(QObject::tr("RGB Bilateral filter"));
+				dlg.setWindowTitle(QObject::tr("RGB Gaussian filter"));
 
 				dlg.doubleSpinBox1->setStatusTip(QObject::tr("3*sigma = 99.7% attenuation"));
-				dlg.doubleSpinBox2->setStatusTip(QObject::tr("It will only use colors for averaging where all of color components in the range of [threshold:255-threshold]"));
+				dlg.doubleSpinBox2->setStatusTip(QObject::tr("It will only use colors for averaging where all of the color components in the range of [threshold:255-threshold]"));
```

Comment:
```suggestion
				dlg.doubleSpinBox2->setStatusTip(QObject::tr("For averaging, it will only use colors for which all components are in the range [threshold:255-threshold]"));
```